src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,16 +16,17 @@
  * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
-
-    public static final String COMMAND_WORD = "find";
-
     public static final String ARGUMENT_WORD = "all";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all employees and potential hires whose names "
-            + "contain any of the specified keywords (case-insensitive) and displays them as a list with "
-            + "index numbers.\n"
-            + "Parameters: (all/e/ph) KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " " + ARGUMENT_WORD + " alice bob charlie";
+    public static final String COMMAND_WORD = "find";
+    public static final String MESSAGE_PURPOSE = "Finds all employees and/or potential hires whose names "
+            + "contain any of the specified keywords (case-insensitive) and displays them as a list with";
```

Comment:
Did you add an additional with?

Reply:
Will fix the typo. Thank you!

Reply:
Did you forget a full stop at the end as well?